helloworld-samples/function-http-nodejs/main.js

```diff
@@ -0,0 +1,13 @@
+module.exports.main = main = async (args) => {
+    const url = 'https://httpbin.org/get';
+    const response = await fetch(url);
+    const data = await response.json();
+
+    return {
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        statusCode: 200,
```

Comment:
How about tying the responseStatus to the rc of the http call to httpbin org?

Reply:
Good Idea 👍